README.md

```diff
@@ -1,28 +1,111 @@
+the same origin, and then the media player and main page can exchange data.
+
+[Setting document.domain has been deprecated]((https://html.spec.whatwg.org/multipage/origin.html#relaxing-the-same-origin-restriction)
+for along time, but continues being supported by browsers.
+This forces process allocation to be by site,
+not by origin, because a page *might* use `document.domain` to change their
+origin - within the site - later on. Measurements indicate that [setting
+`document.domain` happens on around 0.4% of page views](https://chromestatus.com/metrics/feature/timeline/popularity/2544)  So 99% of pages do not even use this feature. But because they *might* want
+to set `document.domain` later on, browsers have to allocate processes by site.
+
+In other words: > 99% of page views pay for a feature - in terms of reduced
+security - that they don't make any use of.
 
 ## A Proposal
 
-Accepting the post-Spectre threat model means that we must begin work in earnest to unblock origin-level process isolation by removing the `document.domain` setter from the platform, sooner rather than later. Given its broad usage, an iterative approach to removal seems reasonable.
-
-We can begin with simple subsetting: the setter's effect on a origin's [domain](https://html.spec.whatwg.org/#concept-origin-domain) should be a no-op in the presence of `Cross-Origin-Opener-Policy` headers, as it already is for `Origin-Isolation` and `crossOriginIsolated` documents.
-
-Following that, we can create a ratcheting deprecation mechanism by introducing a `document-domain` Document Policy configuration point, with a default value of `false` (after an appropriate period of developer-facing messaging in places like browsers developer console, Lighthouse, etc). This policy would similarly disable `document.domain` by default, while still providing developers the ability to opt-into insecurity by sending an explicit `Document-Policy: document-domain` header along with their response. This explicit opt-in will allow us to track usage via HTTP Archive and browser telemetry, and to make appropriate decisions about a document's process when committing a navigation.
-
-As we reduce usage over time, we can strengthen this initial shift by requiring additional opt-in. Enterprise policies and reverse origin trials come to mind as subsequent milestones.
+The [`Òrigin-Agent-Cluster:` http header](`Origin-Agent-Cluster:` header](https://web.dev/origin-agent-cluster/)
+([spec](https://html.spec.whatwg.org/multipage/origin.html#origin-keyed-agent-clusters))
+allows a page to request being isolated by origin (instead of site). If set
+`true`(`Origin-Agent-Cluster: ?1`), the browser is asked to isolate pages by
+origin. (Agent Cluster is spec speak for isolation groups. Since the low-level
+isolation is not visible at the API layer, specifications only cursorily
+touches the subject.) As a side effect, writing to the  document.domain`
+accessor is blocked.
```

Comment:
s/blocked/ignored/? I don't think we throw in this case, which (at least in my head) "blocked" might imply.